### PR TITLE
MTM-45961 Document how the knowledge hub from gainsight can be removed

### PR DESCRIPTION
**Pull request by sag-lukjan**

### CODE REVIEW

File: content/users-guide/getting-started-bundle/knowledge-hub.md [hunk 1]
```diff
@@ -68,3 +68,14 @@ At the bottom of the list in the **Knowledge Hub** dialog, you can find several
 * **UPGRADE {{< product-c8y-iot >}}** - View pricing and purchase {{< product-c8y-iot >}}
 
 These options may vary from time to time.
+
+### Turning off
```
Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/knowledge-hub.md:72]:
```suggestion
### Turning off the Knowledge Hub
```

File: content/users-guide/getting-started-bundle/knowledge-hub.md [hunk 2]
```diff
@@ -68,3 +68,14 @@ At the bottom of the list in the **Knowledge Hub** dialog, you can find several
...
+
+In case you would like to turn off the **Knowledge Hub**:
```
Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/knowledge-hub.md:74]:
```suggestion
To turn off the Knowledge Hub:
```
Knowledge Hub itself is not formatted in bold, only when referring to a screen element (like a button)

File: content/users-guide/getting-started-bundle/knowledge-hub.md [hunk 3]
```diff
@@ -68,3 +68,14 @@ At the bottom of the list in the **Knowledge Hub** dialog, you can find several
...
+
+1. Click the **User** button at the right of the top bar and then click **User Settings**.
+2. In the **Edit user** dialog box click the switch that is called **Enable knowledge hub**.
```
Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/knowledge-hub.md:77]:
```suggestion
2. In the **Edit user** dialog box switch the **Enable Knowledge Hub** toggle.
```
This is misspelled in the UI. Knowledge Hub is always capitalized.

Comment (sag-lukjan) [reply content/users-guide/getting-started-bundle/knowledge-hub.md:77]: Ok :+1:

**Comment by BeateRixen** [comment]
@sag-lukjan How about the UI change. The spelling in the UI must be corrected and the screenshot must be recreated.

**Comment by sag-lukjan** [comment]
> @sag-lukjan How about the UI change. The spelling in the UI must be corrected and the screenshot must be recreated.
Ticket responsible for that feature is already closed: https://cumulocity.atlassian.net/browse/MTM-45735

I'm working on another improvement where functionality of present Knowledge Hub switch will be extended: https://cumulocity.atlassian.net/browse/MTM-48214 so the name will also be changed.

Should I revert those documentation changes?

**Comment by BeateRixen** [comment]
> > @sag-lukjan How about the UI change. The spelling in the UI must be corrected and the screenshot must be recreated.
> > Ticket responsible for that feature is already closed: https://cumulocity.atlassian.net/browse/MTM-45735
> 
> I'm working on another improvement where functionality of present Knowledge Hub switch will be extended: https://cumulocity.atlassian.net/browse/MTM-48214 so the name will also be changed.
> 
> @BeateRixen, should I revert those documentation changes?

I leave this up to you. You can merge these but have to track somewhere that the screenshot needs to be adapted. Or you just keep the PR open for now until you are done. That I would add a label "on hold" to it. Is this for 10.16?

**Comment by sag-lukjan** [comment]
> > > @sag-lukjan How about the UI change. The spelling in the UI must be corrected and the screenshot must be recreated.
> > > Ticket responsible for that feature is already closed: https://cumulocity.atlassian.net/browse/MTM-45735
> > 
> > 
> > I'm working on another improvement where functionality of present Knowledge Hub switch will be extended: https://cumulocity.atlassian.net/browse/MTM-48214 so the name will also be changed.
> > @BeateRixen, should I revert those documentation changes?
> 
> I leave this up to you. You can merge these but have to track somewhere that the screenshot needs to be adapted. Or you just keep the PR open for now until you are done. Then I would add a label "on hold" to it. Is this for 10.16?

I hope that improvement task will be finished before 10.16 release, so maybe I will hold with merging that PR for some time until it's done.

**Comment by sag-lukjan** [comment]
@BeateRixen I've updated a screenshot, because Paweł already fixed an issue with not capitalized "Knowledge Hub" in his pr: https://github.softwareag.com/IOTA/cumulocity-ui/pull/1570#discussion_r73471
I did not have a time to work on https://cumulocity.atlassian.net/browse/MTM-48214, yet, but I think the documentation can be updated.

**Review by BeateRixen — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Pull request merged**